index.js

```diff
@@ -0,0 +1,13 @@
+import './style.css';
+import './jumpy.gif';
+import displayPopup from './src/Module/display_popup.js';
+import displayReservation from './src/Module/display_popup_reservation.js';
+import display from './src/Module/display_home.js';
+import { itemCounter } from './src/Module/counters.js';
+/* eslint-disable no-unused-vars */
+
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] **Optional:** These linters show up because of lines 1 and 2. You have not used the style or jumpy.